DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/ActiveSiteInfoPublisher.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  SiteTroubleshootingInfoPublisher.swift
+//  ActiveSiteInfoPublisher.swift
```

Comment:
The rename is because the old name wasn't really doing a good job of portraying what data this structure was holding.

Reply:
Makes sense, thanks!